tests/cli/test_report.py

```diff
@@ -112,6 +113,23 @@ def test_report_failure():
     Report(filter_args={"tags": "tutorials"}, failure=True)
 
 
+@pytest.mark.cli
+def test_report_start_and_end():
+
+    fmt = "%Y-%m-%d"
+    start_date = datetime.datetime.strptime("2022-06-07", fmt)
+    end_date = datetime.datetime.strptime("2022-06-09", fmt)
+
+    # buildtest report --filter tags=tutorials --start
+    Report(filter_args={"tags": "tutorials"}, start=start_date)
+
+    # buildtest report --filter tags=tutorials --end
+    Report(filter_args={"tags": "tutorials"}, end=end_date)
+
+    # buildtest report --filter tags=tutorials --start --end
+    Report(filter_args={"tags": "tutorials"}, start=start_date, end=end_date)
```

Comment:
This looks good we may also want to test by time. Perhaps add `00:00:00` after the date and use another time stamp or use `datetime.datetime.now()` to do by current time.